packages/cactus-odap-hermes/README.md

```diff
@@ -0,0 +1,50 @@
+# @hyperledger/cactus-odap-hermes
```

Comment:
@jscode017 Please call it `@hyperledger/cactus-plugin-odap-hermes` 
Make sure to search the entire project with free text search for all instances of this name and replace all of them.
Also make sure to rename the directory itself accordingly as well to match the naming convention of the other plugins which are all called `cactus-plugin-*`

Reply:
ok, solve this with new commits